package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dsp-ui-lib",
-  "version": "1.0.0-rc.6",
+  "version": "1.0.0-rc.7",
```

Comment:
Is this necessary (the app won't be published)?

Reply:
No, this is not necessary, you're right. We have always updated this version. But in the new release process this will no longer be relevant anyway. This is still the old way